src/controllers/TeacherController.js

```diff
@@ -57,4 +57,17 @@ module.exports = {
       res.status(400).send(error);
     }
   },
+  async getBySubjectId(req, res) {
+    const { subjectId } = req.query;
+    if (subjectId) {
+      try {
+        const teachers = await Teacher.findAll({
+          where: { subjectId },
+        });
+        return res.status(200).send(teachers);
+      } catch (error) {
+        return res.status(500).send(error);
+      }
+    }
```

Comment:
next function move to next matching route
добавить next() для того, чтобы можно было перейти к следующей функции, т.к. у нас 2 функции на один рут
```suggestion
    }
    next();
```

Reply:
добавил